### DISCUSSION TITLE
Add next/prev track button in voice status bar control

### DISCUSSION THREAD

**Discussion by markusd112** [PaulWoitaschek/Voice#1882] [Ideas]
When playing an audiobook with voice there is a control panel in android status bar visible. There are two buttons to fast forward/backward the current MP3 track, but there are missing the two buttons to go forward/backward one complete track to the next/previous MP3 file.

Please include these 2 buttons.

Please include the following details in your bug report:

App-Version: 4.3.2
Phone Name: Motorola Moto G6plus
Android Version: 9

![Screenshot_20190508-055718](https://user-images.githubusercontent.com/14372683/57447874-54fbfe00-7258-11e9-8d55-d8d079254099.png)

**Comment by innesst** [comment]
How frequently so you need to skip entire chapters? For me it is almost never, and therefore having those extra buttons would create more opportunities for fat finger mistakes than benefits.

**Comment by markusd112** [comment]
I use it everyday: I am using Voice to hear a newspaper (Die ZEIT). Each news article is in a MP3 file and I skip uninteresting articles, so I am missing these buttons. 
Another usecase for me is when I am listening audiobooks for learning a language: I am learning italian language and the school book has included an audiobook CD. I am skipping tracks to go around the lessons to repeat it. Another usecase for me is when I am listening italian audiobooks, that are offered especially for learning the language: there I often skip over/back chapters.

So the two buttons are very helpful for these usecases. To avoid pressing the wrong buttons with fat fingers you could place the skip track buttons on the right side.

<<    ||    >>      .....space.....        |<   >|

But more common is the order of the buttons you can find in the main player windows of voice:

|<   <<     |>    >>      >|

Or you could use different background colors for the fast forward/backward buttons to make it clear where to press for the fat fingers... ;-)

**Comment by KlugB** [comment]
I support this feature request. I'd also like to have an option for the behavior of the app when using the forward/backward button of Bluetooth devices like headset, speakers, car audio system etc. I want to skip chapters more frequently, too.
